kt/godot-library/src/main/kotlin/godot/core/callable/KtCallable.kt

```diff
@@ -17,10 +18,27 @@ abstract class KtCallable<T : KtObject, R : Any?>(
 
     fun invoke(instance: T) {
         TransferContext.readArguments(types, isNullables, paramsArray)
-        val ret = invokeKt(instance)
+        try {
+            invokeKt(instance)
+        } catch (e: Exception) {
```

Comment:
Shouldn't we catch `Throwable` in these instead as these would also catch `RuntimeExceptions` and `Error` instances?

Reply:
RuntimeExceptions are already exception. And regarding Errors, I forgot about them, it's not usually something you recover from, but I can do the change regardless.